Modules/posixmodule.c

```diff
@@ -8720,11 +8724,20 @@ os_pwritev_impl(PyObject *module, int fd, PyObject *buffers, Py_off_t offset,
     }
 
     cnt = PySequence_Size(buffers);
-    if (cnt < 0)
+    if (cnt < 0){
         return -1;
+    }
 
-    if (iov_setup(&iov, &buf, buffers, cnt, PyBUF_SIMPLE) < 0)
+#ifndef HAVE_PWRITEV2
+    if(flags != 0){
+        argument_unavailable_error("pwritev2", "pwritev2() not available in this system");
```

Comment:
You misused the API. It should be something like: argument_unavailable_error("pwritev", "flags");

Reply:
Sorry, my fault. Is 2AM here and I clearly should be sleeping. :sleepy: